Detectors/TRD/reconstruction/src/CruRawReader.cxx

```diff
@@ -542,7 +542,7 @@ int CruRawReader::processHalfCRU(int cruhbfstartoffset)
       ** DIGITS NOW ***
       *****************/
       // Check if we have a calibration trigger ergo we do actually have digits data. check if we are now at the end of the data due to bugs, i.e. if trackletparsing read padding words.
-      if (linkstart != linkend && mCurrentHalfCRUHeader.EventType == o2::trd::constants::ETYPECALIBRATIONTRIGGER) { // calibration trigger
+      if (linkstart != linkend && (mCurrentHalfCRUHeader.EventType == o2::trd::constants::ETYPECALIBRATIONTRIGGER || mOptions[TRDIgnore2StageTrigger])) { // calibration trigger
```

Comment:
So without the 2-stage trigger we always had digits in the raw data? Did you test it with both old and new raw data format?

Reply:
Yes, but only hcheader and trailers, but it gets caught by the linkstart != linkend if there are no digits.

Reply:
sounds good